mitodl/mit-open
# Add ordering to testimonials, adjust view on homepage testimonial carousel

### What are the relevant tickets?

Closes mitodl/hq#4534

### Description (What does it do?)

This adds a few things to the testimonials code:
- You can now specify an order for testimonials. Testimonials are listed in order of specified order, then last updated date when listing them out.
- You can filter testimonials by their position (i.e. if you just want to see things in position 1, you can filter for that).
- Updates the homepage testimonial carousel to only load items in position 1, and then to shuffle what it has loaded so the card display is randomized.

There were a couple of small changes made as well:
- If there aren't any testimonials, the testimonial carousel on the homepage should not display. (This makes it consistent with [the internal carousels](https://github.com/mitodl/mit-open/pull/1001).) 
- The Django admin now shows the testimonials as name, title, position, rather than title, name; makes more sense this way. 

### How can this be tested?

Automated tests should pass.

You will need a set of testimonials to test this manually.  They should have an order and the order should be set according to the offeror (so each offeror that you've tagged has something in position 1, 2, 3, etc.). 

Loading the home page should display only testimonials in position 1 and their order should be randomized.
